### DISCUSSION TITLE
Files in the public directory are served at the root path

### DISCUSSION THREAD

**Discussion by yenche123** [vuejs/devtools#115] [Q&A] [answered]
What is that?
![image](https://github.com/vuejs/devtools-next/assets/28533125/4a3eff06-f666-42c5-8f74-8ed12778845b)

I can make sure they were printed on my VS-Code terminal by vite-plugin-vue-devtools just beacuse I only upgraded it without other dependencies.

Context:
vite-plugin-vue-devtools: 7.0.2

**Comment by alexzhang1030** [comment, marked as answer]
see #117

**Reply by yenche123** [reply]
Thanks a lot!

**Reply by webfansplz** [reply]
Fixed by #117
